pymoose/examples/grpc/README.md

```diff
@@ -1,11 +1,23 @@
 ## Running
 
-Make sure to have three gRPC workers at the right endpoints before running this example.
+Make sure to have three gRPC workers at the right endpoints before running these examples. Moose offers the option to run gRPC workers with in-memory storage or with file storage (accept numpy and csv files).
```

Comment:
Should we just remove the in-memory option? When would it be useful?

Reply:
It's a good point. You right, I think we can remove this option.